UnitySetup/UnitySetup.psm1

```diff
@@ -471,7 +467,7 @@ function Install-UnitySetupInstance {
                 }
             }
 
-            Start-BitsTransfer -Source $downloadSource -Destination $downloadDest
```

Comment:
This is actually downloading multiple objects in one call as `Start-BitsTransfer` can take a list for the source/destination. I think you'll need to update the surrounding context a bit for this code to maybe kick all the downloads and then wait on them or iterate over them and sync download them one at a time.

Reply:
I think it would be good to see if we can determine if Bits is unable to run and have WebClient as the fallback too. Bits does run faster than WebClient, if available.

Reply:
Only looked up a single data point before. Here's the first site I saw: [https://blog.jourdant.me/post/3-ways-to-download-files-with-powershell](https://blog.jourdant.me/post/3-ways-to-download-files-with-powershell)

...though looking back at the numbers here 1 second seems insignificant.

Found another showing multiple other options with different results: [https://powershellone.wordpress.com/2015/10/27/review-of-methods-to-download-files-using-powershell](https://powershellone.wordpress.com/2015/10/27/review-of-methods-to-download-files-using-powershell/)

Reply:
in the end you guys let me know - i agree the sequential nature of downloading would be slower, but i'm not sure how often multiple installers are pulled.. for us it'll always be 1.